gitlab/__init__.py

```diff
@@ -1376,3 +1404,12 @@ def Project(self, id=None, **kwargs):
         return TeamProject._get_list_or_object(self.gitlab, id,
                                                team_id=self.id,
                                                **kwargs)
+
+class ForkProject(GitlabObject):
+    _url = '/projects/fork/%(project_id)s'
+    canUpdate = False
+    canDelete = False
+    canList = False
+    canGet = False
+    requiredUrlAttrs = ['project_id']
+    requiredCreateAttrs = ['project_id']
```

Comment:
requiredCreateAttrs should not be defined. If it exists the CLI will not work (the project-id argument would be defined twice).